deeplabcut/pose_estimation_tensorflow/lib/single_object_tracker.py

```diff
@@ -0,0 +1,116 @@
+"""
+DeepLabCut 2.2 Toolbox (deeplabcut.org)
+© A. & M. Mathis Labs
```

Comment:
I am not a lawyer, but I believe this copyright statement may be problematic.

As far as I know, @ehsainit did this work (started as part of a student project in my lab) in Germany. Under German law, he retains copyright any code he wrote. Therefore, even if he wanted to, he could not legally assign copyright to anyone else. (It may be possible under certain circumstances that typically involve a strong contract negotiated prior to employment that it is possible for companies to own the copyright of work produced by their employees in Germany.) I think OSS projects often say "(C) Copyright 2020, Deep Lab Cut authors" or similar to deal with this issue.

Reply:
Thanks for raising this point as well as your suggestion! 

We are currently thinking about how to address this (e.g. [WIP here](https://github.com/DeepLabCut/DeepLabCut/pull/746), your thoughts would be very welcome). My goals are to a) acknowledge all contributors b) keep DLC open-source and in accordance to its license, which means that [contributors should waive their copyright](https://opensource.guide/legal/) so that others can use, modify, and share the work.

Reply:
I think "waiving" copyright is not the correct phrasing. Copyright is the legal means by which open source is enforced. Keeping the code copyrighted (even by many people) is how the license can be enforced. Waiving copyright seems counter to that goal (and, as far as I understand it, not possible under German law).

I think you want the contributor (copyright holder) to license the code under the DLC license. I didn't read your Individual Contributor License Agreement v2.0 in detail, but I think this is exactly what it does. My suggestion is just to change the header from "© A. & M. Mathis Labs" to "© DeepLabCut authors" (or similar) and then have an authors file that all contributors get added to and acknowledges their work. (You may already have this file, I didn't check.) They hold their respective copyrights but grant the provisions specified in the CLA which indeed keep DLC open source in accordance with its license.

Take all this with a grain of salt, however. I am not a lawyer.

Reply:
Yes, I also think that Contributor License Agreement v2.0 does just that. 

I like the suggestion regarding "© DeepLabCut authors". We do have an [authors file, in which we highlight individual contributions](https://github.com/DeepLabCut/DeepLabCut/blob/master/AUTHORS) that should be expanded.

Reply:
Looks good to me, thanks!